write-feed/src/main.ts

```diff
@@ -29,12 +29,21 @@ const run = async ({
   const signer = stripHexPrefix(signerString)
   const topic = bee.makeFeedTopic(topicString)
 
-  const writer = bee.makeFeedWriter('sequence', topic, signer)
-  const response = await writer.upload(postageBatchId, reference)
-  const manifest = await bee.createFeedManifest(postageBatchId, 'sequence', topic, signerToAddress(signer))
+  try {
+    const writer = bee.makeFeedWriter('sequence', topic, signer)
+    const response = await writer.upload(postageBatchId, reference)
+    const manifest = await bee.createFeedManifest(postageBatchId, 'sequence', topic, signerToAddress(signer))
 
-  core.setOutput('reference', response)
-  core.setOutput('manifest', manifest)
+    core.setOutput('reference', response)
+    core.setOutput('manifest', manifest)
+  } catch (err) {
+    if (err instanceof BeeResponseError && err.status === 409) {
```

Comment:
@AuHau is this sufficient condition? I remember we had issues with “instanceof” checks in past.

Reply:
As an example (not the same endpoint, but should be basically the same without the validation):

```js
const { Bee, BeeResponseError } = require('@ethersphere/bee-js')
const bee = new Bee('https://httpstat.us/409')

;(async () => {
	try {
		await bee.pin(Array.from({ length: 64 }).map(i => 0).join(''))
	}
	catch (err) {
		console.assert(err instanceof BeeResponseError, 'err instanceof BeeResponseError')
	}
})()

```

Reply:
Yes but we used to have a problem where any Error was instanceof BeeError. This is due to some nodejs inheritance bug. I wanted to test this in the github action environment.

Reply:
See https://github.com/ethersphere/bee-js/issues/204 but looks like it may be solved (we no linger use webpack for nodejs builds).

Reply:
I added an assert for `new Error instanceof BeeResponseError` which seems to work, but sure, probably best to wait for Adam. Looks like it's also used in `bee-js` directly: https://github.com/ethersphere/bee-js/blob/88039bb5f86e6cf528a9572e92d0146f398b3fb3/test/utils.ts#L291.

Reply:
It should be safe to use in this context. 

The mentioned problem was related to NodeJS modules and especially thrown `TypeErrors` from NodeJS modules that is not the case here. 

Good point @vojtechsimetka that it might not even be in the scope anymore since we dropped webpack nodejs builds. I should re-investigate later on.